pkg/streamingpromql/operators/functions/range_vectors_test.go

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// Provenance-includes-location: https://github.com/prometheus/prometheus/blob/main/promql/functions.go
+// Provenance-includes-license: Apache-2.0
+// Provenance-includes-copyright: The Prometheus Authors
+
+package functions
+
+import (
+	"testing"
+
+	"github.com/prometheus/prometheus/promql"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_runDoubleExponentialSmoothing(t *testing.T) {
+	type args struct {
+		fHead           []promql.FPoint
+		fTail           []promql.FPoint
+		smoothingFactor float64
+		trendFactor     float64
+	}
+	tests := []struct {
+		name         string
+		args         args
+		wantFloat    float64
+		wantHasFloat bool
+		wantErr      bool
```

Comment:
`wantErr` is never used - should we just remove it?

Reply:
Removed 18188b0bd8009f1cb4185d62e35ccd967c6a0b13